redo.dev/docs/model/demos/stackblitzGenerators/createStackblitzDemo.ts

```diff
@@ -1,40 +1,34 @@
 import sdk from "@stackblitz/sdk"
 
+export type EmbedId = "model" | "space" | "constraints" | "declaration"
+export type DeclarationFiles = "user" | "group"
 export type DemoProps = {
-    embedId: "model" | "space" | "declaration" | "constraints"
-    elementId: string
+    embedId: EmbedId
+    addonFiles?: DeclarationFiles[]
```

Comment:
Great solution to this! 😊 In case we want to reuse it for other static files, can we call the type `AddonFile` as well (in general, unions like this should not be plural since it's still only a single string, not a list)?